spotify_player/Cargo.toml

```diff
@@ -4,7 +4,7 @@ version = "0.15.0"
 authors = ["Thang Pham <phamducthang1234@gmail.com>"]
 edition = "2021"
 license = "MIT"
-description = "A command driven spotify player"
+description = "A command driven Spotify player"
```

Comment:
can you update the description to match the Github description as well?

Reply:
Done!